ignite/services/chain/plugin-stargate.go

```diff
@@ -66,35 +66,37 @@ func (p *stargatePlugin) appTOML(homePath string, cfg *chainconfig.Config) error
 		return err
 	}
 
-	validator := cfg.Validators[0]
-	servers, err := validator.GetServers()
-	if err != nil {
-		return err
-	}
+	if cfg.Validators != nil {
```

Comment:
Instead of adding these conditions in `appTOML` and `configTOML` I would prefer to make a refactoring to skip `appTOML` and `configTOML` when the workflow is called from `Network`

Because we should not modify any information in `app.toml` or `config.toml` like
```
config.Set("api.enable", true)
config.Set("api.enabled-unsafe-cors", true)
config.Set("rpc.cors_allowed_origins", []string{"*"})
```
for example